app/components/code-snippet.js

```diff
@@ -39,10 +39,12 @@ export default Ember.Component.extend({
   }),
 
   didInsertElement: function(){
-    Highlight.highlightBlock(this.get('element'));
+    // Highlight.highlightBlock(this.get('element'));
```

Comment:
I had hacked this in which worked for our use case. Once you have `Prism` on the page you can do something like the following:

``` javascript
const Prism = self.Prism;
let codeElement = this.get('element').querySelector('code');
let highlighted = Prism.highlight(codeElement.innerText.trim(), Prism.languages[this.get('language')], this.get('language'));
codeElement.innerHTML = highlighted;
```

This way if you have a tabbed UI like in ember-cli-addons-docs the syntax highlighting will still work as you switch between the tabs.

Reply:
there is also
```ts
Prism.highlightAllUnder(this.element);
```

Reply:
Oh TIL 🙌